Propose new definition for Guys. (#259)
tatianamac/selfdefined: 11ty/definitions/guys.md

---

> title: Guys

Reviewer: ```suggestion
title: guys
```

slug: guys
defined: false
speech: noun
reading:
  - text: Hey Guys
    href: https://heyguys.cc/
  - text: Online etymology dictionary
    href: https://www.etymonline.com/word/guy
alt_words:
  - all
  - comrades
  - everyone
  - folks
  - friends
  - peeps
  - people
  - team
  - y'all
  - yinz
---

> This term is used colloquial for a person (guy, singular) or group of people (guys, plural); its origins come from Guy Fawkes.

Reviewer: ```suggestion
colloquially, a group of people (or, a person when singular); its origin comes from Guy Fawkes, a man, and thus, historically referenced a man or men/fellow or fellows. 
```

> "Hey guys" is a common salutation, but it's not gender neutral. People who aren't men can feel excluded when addressed as "guys," even if that is not the intent.

Reviewer: ```suggestion
## Issues
"Hey guys" is a common English salutation for groups of people. However, as guys is not gender-neutral, people can feel excluded when addressed as "guys."

## Impact
"Guys" is unnecessarily gendered language. Gendered language often reinforces the gender binary, which excludes non-binary people. Even within the gender binary, gendered language tends to favour men, (e.g., congressmen, mankind, etc) which can reinforce that people who don't identify as men do not belong in these spaces, particularly when the data shows that they are already mostly comprising men and dominated by men (who represent the leadership, for one example).

Additionally, for many trans women, people who are trans non-binary, agender, and/or gender non-conforming in particular, it can be triggering to be referred to as a "guy."

## Avoiding Monoliths
While some people who don't identify as men are not bothered by being called guys when they don't identify as men, they do not represent all people in their groups. We should be working to reduce harm, not establish universal consensus. Plenty of more inclusive alternates exist that avoid gendered language altogether.
```
